app/jobs/code_worker.rb

```diff
@@ -8,18 +8,53 @@
 
 class CodeWorker
   include SuckerPunch::Job
-
+  include TestCaseHelper
   # Reducing to 2 workers, since it seems that each puma process will
   # have its own job queue and its own set of sucker punch worker threads.
   # We'll get parallelism through puma processes instead of sucker punch
   # workers.
   workers 2 # 10
 
+  def self.parse_java(string)
```

Comment:
`parse_java` doesn't say much about what this method does. Rename to something that says it grabs student-written test cases from the attempt string. (The param name `string` should be improved as well)